### PR TITLE
fix(docs): typos. Copy edited.

### PR DESCRIPTION
**Pull request by the-sheev**

### CODE REVIEW

**Review by bsmth — DISMISSED** [review]

**Review by bluestreak01 — APPROVED** [review]

File: docs/concept/designated-timestamp.md [hunk 1]
```diff
@@ -73,48 +73,48 @@ There are two approaches that can be used in such cases:
...
-  ```
-
-  ```questdb-sql title="Using system timestamp while retaining the device timestamp"
-  INSERT INTO readings VALUES(
-      systimestamp(),
-      to_timestamp('2020-03-01:15:43:21', 'yyyy-MM-dd:HH:mm:ss'),
-      'my_sensor',
-      123);
-  ```
+```questdb-sql title="The db_ts column is specified as designated timestamp"
```
Comment (bsmth) [inline docs/concept/designated-timestamp.md:76]: Maybe we could wrap these code blocks in `prettier-ignore` tags and put the indentation back, as actually this is an OK way to nest the code blocks in a list item

File: blog/2020-03-15-interthread.md [hunk 2]
```diff
@@ -18,7 +18,12 @@ tags: [deep-dive]
 
 import Banner from "@theme/Banner"
 
-<Banner alt="Flock of birds flying in a harmonised way" height={360} src="/img/blog/2020-03-15/header.png" width={650} />
+<Banner
+  alt="Flock of birds flying in a harmonised way"
+  height={360}
+  src="/img/blog/2020-03-15/header.png"
+  width={650}
+/>
```
Comment (bsmth) [inline blog/2020-03-15-interthread.md:26]: As discussed in DM, prettier does not produce the same result for me when running on this file

Comment (bsmth) [reply blog/2020-03-15-interthread.md:26]: ran a fresh clone and this issue is gone for me

**Review by bsmth — APPROVED** [review]
thanks 👍🏻

**Review by bsmth — APPROVED** [review]

**Pull request merged**